docroot/themes/humsci/humsci_basic/src/scss/components/_global-message.scss

```diff
@@ -0,0 +1,125 @@
+// This is a stylesheet for the Global Message banner
+.su-alert {
+  background-color: #f4f4f4;
+  color: #2e2d29;
+  padding: 0;
+
+  & .fa,
+  & .fas,
+  .hb-dark-pattern & .fa,
+  .hb-dark-pattern & .fas {
+    color: #2e2d29;
```

Comment:
Let's update all the color values to use Decanter variables were possible. For example, this would be `color: $su-color-black`. Here's a link to Decanter's color documentation: https://decanter.stanford.edu/page/brand-design-elements-color/

Reply:
Great catch, these have been updated but we only have decanter variables for black, red and blue.  The others have been left as hex colors.